components/markdown/lists.tsx

```diff
@@ -0,0 +1,84 @@
+import Container from './container';
+import styled from 'styled-components';
+import css from '@styled-system/css';
+
+const StyledUl = styled.ul`
+  ${css({
+    pl: '550',
+    m: '0',
+    mb: '450',
+  })}
+
+  li p, li > div {
+    ${css({
+      p: '0',
+      m: '0',
+    })}
+  }
+
+  li {
+    ${css({
+      pl: '0',
+      lineHeight: '300',
+    })}
+    position: relative;
+    list-style-type: none;
+  }
+
+  li:before {
+    display: block;
+    content: '';
+    height: 5px;
+    width: 5px;
+    position: absolute;
+    left: -18px;
+    border-radius: 50%;
+    top: 9px;
+
+    ${css({
+      bg: 'gray.1000',
+    })}
+  }
+`;
+
+type UlProps = {
+  children?: React.ReactNode;
+};
+
+export const Ul = (props: UlProps): JSX.Element => {
+  return (
+    <Container>
+      <StyledUl>{props.children}</StyledUl>
+    </Container>
+  );
+};
+
+const StyledOl = styled.ol`
+  ${css({
+    pl: '550',
+    m: '0',
+    mb: '450',
+  })}
+
+  li p, li > div {
+    ${css({
+      p: '0',
+      m: '0',
+    })}
+  }
+
+  li {
+    ${css({
+      pl: '0',
+      lineHeight: '300',
+    })}
+  }
+`;
```

Comment:
Design has `<ol>`/`<ul>` in the same group without giving an example of an `<ol>`, but looking at the way `<ul>` is set, I'd like to see the `<ol>` more centered, giving the number a little more "call out" and easier to read apart from the text.

Since the numbers are ~16px wide, basically centering the value inside of the 27px space a little nicer
<img width="1309" alt="Screen Shot 2021-11-05 at 2 50 58 PM" src="https://user-images.githubusercontent.com/82669192/140583217-9a0d3b03-b278-413c-9a9b-1eae8f96bd71.png">

Reply:
Also, do we want to increase the `font-weight` of the numbers? Don't have to, but want to ask.